base/compiler/tfuncs.jl

```diff
@@ -1362,6 +1362,27 @@ add_tfunc(arrayref, 3, INT_INF, arrayref_tfunc, 20)
 add_tfunc(const_arrayref, 3, INT_INF, arrayref_tfunc, 20)
 add_tfunc(arrayset, 4, INT_INF, (@nospecialize(boundscheck), @nospecialize(a), @nospecialize(v), @nospecialize i...)->a, 20)
 
+function _opaque_closure_tfunc(@nospecialize(arg), @nospecialize(isva),
+        @nospecialize(lb), @nospecialize(ub), @nospecialize(source), env::Vector{Any},
+        linfo::MethodInstance)
```

Comment:
Alternatively, this could be written more compactly as. Thoughts?
```suggestion
function _opaque_closure_tfunc(arg::Any, isva::Any, lb::Any, ub::Any, source::Any,
        env::Vector{Any}, linfo::MethodInstance)
    @nospecialize
```

Reply:
`@nospecialized` is declared at [the top of this file](https://github.com/JuliaLang/julia/blob/310ac381934b381a76daf4c6d629e1f17361087f/base/compiler/tfuncs.jl#L7), so I guess we even don't need these annotations in tfuncs.jl ?

Reply:
I like the explicit nospecialize annotations. Otherwise I always forget to put them back